code/modules/modular_computers/computers/item/pda.dm

```diff
@@ -67,14 +67,14 @@
 		var/datum/computer_file/program/program_type = new programs
 		store_file(program_type)
 
-/obj/item/modular_computer/pda/update_overlays()
+/obj/item/modular_computer/pda/update_overlays(ui_overlay)
```

Comment:
What's the point of this parameter exactly? Is this proc called somewhere where it passes this?

I don't really think you should just add parameters to this, in any case